packages/manager/modules/pci/src/projects/project/private-networks/private-networks.html

```diff
@@ -67,49 +66,106 @@
-            data-searchable
             data-filterable
         ></oui-datagrid-column>
-        <oui-datagrid-column
-            data-title=":: 'pci_projects_project_network_private_address' | translate"
-            property="address"
-        ></oui-datagrid-column>
-        <oui-action-menu data-compact data-placement="end">
-            <oui-action-menu-item on-click="$ctrl.deleteNetwork($row.id)">
-                <span
-                    data-translate="pci_projects_project_network_private_delete"
-                ></span>
-            </oui-action-menu-item>
-        </oui-action-menu>
+        <oui-datagrid-row-detail>
+            <div
+                class="oui-table-responsive grid-details-table"
+                data-ng-if="$row.subnet.length > 0"
+            >
+                <table class="oui-table">
+                    <thead class="oui-table__headers">
+                        <tr>
+                            <th scope="col" class="oui-table__header"></th>
+                            <th scope="col" class="oui-table__header"></th>
+                            <th
+                                scope="col"
+                                class="oui-table__header"
+                                data-ng-bind="'pci_projects_project_network_private_region' | translate"
+                            ></th>
+                            <th
+                                scope="col"
+                                class="oui-table__header"
+                                data-ng-bind="'pci_projects_project_network_private_address' | translate"
+                            ></th>
+                            <th
+                                scope="col"
+                                class="oui-table__header"
+                                data-ng-bind="'pci_projects_project_network_private_gateway' | translate"
+                            ></th>
+                            <th
+                                scope="col"
+                                class="oui-table__header"
+                                data-ng-bind="'pci_projects_project_network_private_dhcp' | translate"
+                            ></th>
+                            <th
+                                scope="col"
+                                class="oui-table__header"
+                                data-ng-bind="'pci_projects_project_network_private_ip_allocation' | translate"
+                            ></th>
+                            <th scope="col" class="oui-table__header"></th>
+                        </tr>
+                    </thead>
+                    <tbody class="oui-table__body">
+                        <tr
+                            class="oui-table__row"
+                            data-ng-repeat="subnetDetail in $row.subnet track by $index"
+                        >
+                            <td class="oui-table__cell"></td>
+                            <td class="oui-table__cell">Test PN</td>
```

Comment:
<td> data cell value is hardcoded here ?

Reply:
removed the hardcoded value.